recipes/xinvert/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xinvert" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7817a1604d0d122c3ba2d9457396ce99f52f3d6ca7b8db7fd78d28d30358a70b
+
+build:
```

Comment:
It seems like this package could probably be noarch. Can you please check?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds
Easier way to generate the package: https://github.com/conda/grayskull